src/components/PopupWithForm.js

```diff
@@ -2,19 +2,19 @@ import Popup from "./Popup.js";
 export default class PopupWithForm extends Popup {
   #handleFormSubmit;
   #modalForm;
-
+  #modalSaveButton;
   #modalInputs;
   constructor(popupSelector, handleFormSubmit) {
     super(popupSelector);
 
     this.#handleFormSubmit = handleFormSubmit;
     this.#modalForm = this._modal.querySelector(".modal__form");
+    this.#modalSaveButton = this.#modalForm.querySelector(
+      ".modal__button-save"
+    );
     this.#modalInputs = this.#modalForm.querySelectorAll(".modal__form-input");
   }
-  close() {
-    super.close();
-    this.#modalForm.reset();
-  }
```

Comment:
You need to have method `close` inside `PopupWithForm` where you need to reset the form according to the task